### PR TITLE
Add usage example to tf.image.pad_to_bounding_box

### PR DESCRIPTION
**Pull request by RichardXiao13**

### CODE REVIEW

**Comment by RichardXiao13** [comment]
@googlebot I signed it!

File: tensorflow/python/ops/image_ops_impl.py [hunk 1]
```diff
@@ -788,6 +788,13 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
     If `image` was 3-D, a 3-D float Tensor of shape
     `[target_height, target_width, channels]`
 
+Usage Example:
+    '''python
+    import tensorflow as tf
+    x = tf.random.normal(shape=(256, 256, 3))
+    x = tf.image.pad_to_bounding_box(x, 2, 2, 260, 260)
```
Comment (alextp) [inline tensorflow/python/ops/image_ops_impl.py]: Use >>> to make this a doctest, do not import tensorflow, and print the shape after padding

File: tensorflow/python/ops/image_ops_impl.py [hunk 2]
```diff
@@ -789,11 +789,9 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
...
 
 Usage Example:
-    '''python
-    import tensorflow as tf
-    x = tf.random.normal(shape=(256, 256, 3))
-    x = tf.image.pad_to_bounding_box(x, 2, 2, 260, 260)
-    '''
+>>> x = tf.random.normal(shape=(256, 256, 3))
+>>> x = tf.image.pad_to_bounding_box(x, 2, 2, 260, 260)
+<x.shape = TensorShape([Dimension(260), Dimension(260), Dimension(3)])>
```
Comment (alextp) [inline tensorflow/python/ops/image_ops_impl.py]: It should read TensorShape([260, 260, 3]); did you run the example with tf1?

Comment (RichardXiao13) [reply tensorflow/python/ops/image_ops_impl.py]: That seemed to be the problem. I have fixed the issue.

**Review by alextp — DISMISSED** [review]

File: tensorflow/python/ops/image_ops_impl.py [hunk 3]
```diff
@@ -788,6 +788,11 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
     If `image` was 3-D, a 3-D float Tensor of shape
     `[target_height, target_width, channels]`
 
+Usage Example:
+>>> x = tf.random.normal(shape=(256, 256, 3))
+>>> x = tf.image.pad_to_bounding_box(x, 2, 2, 260, 260)
+<x.shape = TensorShape([260, 260, 3])>
+
```
Comment (rthadur) [inline tensorflow/python/ops/image_ops_impl.py]: @RichardXiao13 this failed the doctest , here is the error : 
` Failed doctest test for google3.third_party.tensorflow.python.ops.image_ops_impl.pad_to_bounding_box
 File "/google3/runfiles/google3/third_party/tensorflow/python/ops/image_ops_impl.py", line 767, in pad_to_bounding_box

----------------------------------------------------------------------
File "/runfiles/google3/third_party/tensorflow/python/ops/image_ops_impl.py", line 793, in google3.third_party.tensorflow.python.ops.image_ops_impl.pad_to_bounding_box
Failed example:
   	x = tf.image.pad_to_bounding_box(x, 2, 2, 260, 260)
Expected:
 	  <x.shape = TensorShape([260, 260, 3])>
`
can you please run the doctest locally and try?

**Review by rthadur — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/image_ops_impl.py (hunk 3 above)
Comment (RichardXiao13) [reply tensorflow/python/ops/image_ops_impl.py]: It seemed that I formatted the output incorrectly. It should work now!

**Comment by rthadur** [comment]
@RichardXiao13 thank you, it is still failing doctest can you please check here for [logs](https://source.cloud.google.com/results/invocations/17549807-4074-4d49-bc57-37698b3250d3/targets/%2F%2Ftensorflow%2Ftools%2Fdocs:tf_doctest/tests)

Please run the doctest locally as mentiones here in the [contributor guidelines.](https://www.tensorflow.org/community/contribute/docs_ref)

**Comment by RichardXiao13** [comment]
@rthadur I am having some trouble getting the doctest to run on my machine. I have been trying for many days to no avail. I am getting an import error message stating 
"
Traceback (most recent call last):
File "tf_doctest.py", line 31, in <module>
from tensorflow.tools.docs import tf_doctest_lib
ImportError: cannot import name 'tf_doctest_lib' 
"
 I am not quite sure how to resolve this.

**Comment by RichardXiao13** [comment]
Actually, I just figured it out!

**Review by alextp — DISMISSED** [review]

**Comment by rthadur** [comment]
@RichardXiao13 thank you for fixing ,we see some sanity checks failing , can you please check.

**Comment by RichardXiao13** [comment]
Just updated it!

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Please use fixed data, not random one and then check that in the output. Make the test deterministic

**Comment by gbaned** [comment]
@RichardXiao13 Can you please check mihaimaruseac's comments and keep us posted? Thanks!

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**